samples/csharp_dotnetcore/14.nlp-with-orchestrator/NLP-With-Orchestrator.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.14.1" />
     <PackageReference Include="Microsoft.Bot.Builder.AI.Orchestrator" Version="4.14.1" />
     <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.14.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.14.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.15.0-daily.20211006.271232.c20cc39" />
```

Comment:
I think we have to update this, maybe not now, but once the 4.15.x is released, related packages will be updated as well.